mongoose.h

```diff
@@ -2776,12 +2778,14 @@ struct mg_tcpip_if {
 #define MG_TCPIP_STATE_DOWN 0   // Interface is down
 #define MG_TCPIP_STATE_UP 1     // Interface is up
 #define MG_TCPIP_STATE_REQ 2    // Interface is up and has requested an IP
```

Comment:
STATE_REQ and STATE_IP are confusing names, IMO.
How about renaming them into `STATE_REQUEST_IP` and `STATE_REQUEST_GATEWAY_MAC` ?

Reply:
Clarified, REQ belongs to the "REQUESTING" state in the DHCP state machine